packages/design-system/src/components/RangeSlider/RangeSlider.interface.ts

```diff
@@ -2,6 +2,10 @@ export interface RangeSliderProps {
   step: number;
   value: IRange;
   labelPosition: TLabelPosition;
+  slidersAriaLabels: {
+    minValueSliderAriaLabel: string;
+    maxValueSliderAriaLabel: string;
+  };
```

Comment:
```suggestion
    minValueSliderAriaLabel?: string;
    maxValueSliderAriaLabel?: string;
```
I agree with @kielbasa-elp we shouldn't make ariaLabels obligatory.